packages/ra-core/src/form/useUnique.ts

```diff
@@ -0,0 +1,112 @@
+import merge from 'lodash/merge';
+import { useResourceContext } from '../core';
+import { useDataProvider } from '../dataProvider';
+import { useTranslate, useTranslateLabel } from '../i18n';
+import { InputProps } from './useInput';
+
+/**
+ * A hook that returns a validation function checking for a record field uniqueness
+ * by calling the dataProvider getList function with a filter.
+ *
+ * @example Passing options at declaration time
```

Comment:
```suggestion
 * @example // Passing options at declaration time
```
Otherwise the syntax highlighting in the IDE fails. 

Same for the other examples